frontend/src/routes/MonthlyWorkingHours.jsx

```diff
@@ -193,19 +194,19 @@ const MonthlyWorkingHours = () => {
           }
         />
       </Col>
-      <Col span={19} push={5}>
-        <Table columns={columns} dataSource={tableData} />{" "}
-      </Col>
-      <Col span={5} pull={19}>
+      <Col style={{ marginTop: "2%" }} span={5} pull={19}>
         Ajanjakso
         <SideBar
           onYearChange={onYearChange}
           selectedYear={selectedYear}
           applyFilters={applyFilters}
-          selectedYearsBack={selectedYearsBack}
           setSelectedYearsBack={setSelectedYearsBack}
+          selectedYearsBack={selectedYearsBack}
         />
       </Col>
+      <Col span={19} push={5}>
+        <Table columns={columns} dataSource={tableData} />{" "}
+      </Col>
```

Comment:
I think you could get the same layout with one column (left side col) that contains the side bar and another column that contains both the bar chart and the table (right side col). But, then you may need another container for the barchart to limit its height (maybe a row)